app/controllers/users_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def show
```

Comment:
- [ ] user controller # index displays all users

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone3/userctr.png)

Just implement the user index and ignore the friendship functionality until friendships milestone.